src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Dto/RemoteVettingIdenityProviderDto.php

```diff
@@ -48,11 +55,17 @@ public static function create(array $configData)
         Assert::allString($configData['description'], 'All description entries must be of type string');
         Assert::notBlank($configData['logo'], 'The logo of a remote vetting identity provider must not be blank');
         Assert::string($configData['logo'], 'The logo of a remote vetting identity provider must be of type string');
+        Assert::string($configData['slug'], 'The slug of a remote identity provider must be of type string');
+        Assert::regex($configData['slug'], self::$slugValidationRegex, sprintf(
+            'The slug of a remote identity provider must validate against regex: %s',
+            self::$slugValidationRegex
+        ));
```

Comment:
Now that validation becomes more involved. It migth be a good time to add a unit test for this Dto!

Reply:
Removed the regex because it was more a helper to use a sane slug in the configuration. Because it's not a functional requirement I've changed it to a simple assertion. That way we could omit the test because there isn't any extensive logic involved.

Reply:
I thought that was a very nice addition. It also helps the user to configure the application with URL safe settings.